src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int FIX_RADIUS = 10;
+    private static final int MAX_NUMBER = 5;
+    private static final int MAX_PARAMETR = 10;
+    private static final String COLOR_WHITE = Color.WHITE.name();

```

Comment:
name of a constant should describe not its value, but its role
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name();

```